### PR TITLE
Feat/mkdocs material

### PR DESCRIPTION
**Pull request by philtom-ctds**

### CODE REVIEW

File: deploy-gdc-docs.sh [hunk 1]
```diff
@@ -73,59 +84,47 @@ if $hasEncodingError  ; then
...
-   /usr/local/bin/mkdocs2pandoc -f ${userGuide}_UG.yml -o docs/${userGuide}/PDF/${userGuide}_UG.pd
-   echo "$(date +'%d %B %Y - %k:%M'): ${ENVIRONMENT}: ${userGuide}: Replacing strings in pandoc document "
-   /bin/sed -i -e 's/# / /g' docs/${userGuide}/PDF/${userGuide}_UG.pd
-   /bin/sed -i -e 's/### /## /g' docs/${userGuide}/PDF/${userGuide}_UG.pd
-   /bin/sed -i -e 's/\/site\//\/docs\//g' docs/${userGuide}/PDF/${userGuide}_UG.pd
-   /bin/sed -i -e "s/(images/(https:\/\/gdc-docs.nci.nih.gov\/"$userGuide"\/Users_Guide\/images/g" docs/${userGuide}/PDF/${userGuide}_UG.pd #To make images clickable in the PDF
-   echo "$(date +'%d %B %Y - %k:%M'): ${ENVIRONMENT}: ${userGuide}: Building PDF from pandoc document "
-   /usr/bin/pandoc --listings -H theme/latex/listings-setup.tex --toc -V documentclass=report -V geometry:"top=2cm, bottom=1.5cm, left=1cm, right=1cm" -f markdown+grid_tables+table_captions docs/${userGuide}/PDF/${userGuide}_Title.txt -o docs/${userGuide}/PDF/${userGuide}_UG.pdf docs/${userGuide}/PDF/${userGuide}_UG.pd
-done
+# Uncomment below - Bilal
```
Comment (bilalbaqar) [inline deploy-gdc-docs.sh:87]: This is temporary. will update.

File: deploy-gdc-docs.sh [hunk 2]
```diff
@@ -73,59 +84,47 @@ if $hasEncodingError  ; then
...
+# done
 
 echo "$(date +'%d %B %Y - %k:%M'): ${ENVIRONMENT}: Cleaning previous website directory (rm)"
 sudo rm /var/www/gdc-docs-${ENVIRONMENT}.nci.nih.gov/* -R
+   # Added by ray 2018-07-11 since it was erroring trying to find a directory ?
+   sleep 10s
 
 echo "$(date +'%d %B %Y - %k:%M'): ${ENVIRONMENT}: Build Encyclopedia"
-python buildencyclopedia.py
+#python buildencyclopedia.py
```
Comment (bilalbaqar) [inline deploy-gdc-docs.sh:118]: This is temporary. Will update.

File: deploy-gdc-docs.sh [hunk 3]
```diff
@@ -100,32 +98,26 @@ for userGuide in "${userGuides[@]}"; do
...
-   echo "$(date +'%d %B %Y - %k:%M'): ${ENVIRONMENT}: ${userGuide}: Building PDF from pandoc document "
-   /usr/bin/pandoc --listings -H theme/latex/listings-setup.tex --toc -V documentclass=report -V geometry:"top=2cm, bottom=1.5cm, left=1cm, right=1cm" -f markdown+grid_tables+table_captions docs/${userGuide}/PDF/${userGuide}_Title.txt -o docs/${userGuide}/PDF/${userGuide}_UG.pdf docs/${userGuide}/PDF/${userGuide}_UG.pd
+   echo "$(date +'%d %B %Y - %k:%M'): ${ENVIRONMENT}: ${userGuide}: Building PDF documents"
+   ENABLE_PDF_EXPORT=1 venv/bin/mkdocs build -f ${userGuide}_UG.yml
 done
 
 echo "$(date +'%d %B %Y - %k:%M'): ${ENVIRONMENT}: Cleaning previous website directory (rm)"
 sudo rm /var/www/gdc-docs-${ENVIRONMENT}.nci.nih.gov/* -R
+   # Added by ray 2018-07-11 since it was erroring trying to find a directory ?
+   sleep 10s
```
Comment (bilalbaqar) [inline deploy-gdc-docs.sh:108]: I will look into the need for this. Seems unnecessary (Famous last words).

**Review by wwysoc2 — APPROVED** [review]
I'm going to approve and merge this, since everything I can understand seems to check out and I think this is what is deployed to production.  Once it's merged, I'm going to try to deploy to the dev site and I'll start asking around if there are any problems.

**Pull request merged**